package.json

```diff
@@ -13,7 +13,7 @@
   "bugs": {
     "url": "https://github.com/codecov/vscode/issues"
   },
-  "version": "0.0.6",
+  "version": "0.0.0",
```

Comment:
this field must be present for `vsce` to work, but we don't actually use the version here. We take it from the release's tag name - so this is just a placeholder. If you have a better idea for version management in mind I'd love to hear it!

Reply:
That seems reasonable to me.